assets/agenda/components/CoverageItemStatus.tsx

```diff
@@ -132,7 +132,7 @@ export default class CoverageItemStatus extends React.Component<any, any> {
                         key="contentLink"
                         className='nh-button nh-button--small nh-button--tertiary nh-button--disabled'
                         ref={(elem) => this.elem = elem}
-                        title={gettext('“You don’t have access to this content, please contact test@localhost”')}
+                        title={gettext('You don’t have access to this content, please contact {{email}}', {email: getConfig('view_content_tooltip_email')})}
```

Comment:
I wonder if this should be actually link with mailto, otherwise it would be hard to get the email from the tooltip

Reply:
yep, adding a clickable `mailto:` for the email address would enhance user experience. However, since the ticket does not specify this requirement.